webapp/packages/core-authentication/src/UserInfoResource.ts

```diff
@@ -81,8 +89,8 @@ export class UserInfoResource extends CachedDataResource<UserInfo | null, void,
     });
   }
 
-  isAuthenticated(): this is { data: UserInfo } {
-    return !this.isAnonymous;
+  isData(): this is { data: UserInfo } {
+    return !!this.data;
   }
```

Comment:
kinda strange, what difference with `if (resource.data)` and `if (resource.isData())`?